src/components/Footer.jsx

```diff
@@ -20,6 +21,9 @@ const Background = styled.div`
   color: #052446;
   text-align: center;
   margin-top: -20%;
+  @media only screen and (max-width: ${TABLET}) {
```

Comment:
Nit: should use the media queries provided by our theme provider, so something like `{p => p.theme.mediaQueries.tablet} ...`